bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java

```diff
@@ -2511,6 +2511,16 @@ public Location getSpectatorLocation() {
         return spectatorLocation;
     }
 
+    @Override
+    public void setAllowMapBreak(boolean value) {
+        getConfig().set(ConfigPath.ARENA_ALLOW_MAP_BREAK, value);
```

Comment:
Im not sure if setting this vallue will also make it take affect immediately

Reply:
Yes this will take effect inmediately, ive tested it with an addon